BLE_SM/source/main.cpp

```diff
@@ -159,9 +162,19 @@ class SMDevice : private mbed::NonCopyable<SMDevice>,
             return;
         }
 
+        /* This path will be used to store bonding information but will fallback
+         * to storing in memory if file access fails (for example due to lack of a filesystem) */
+        const char* db_path = "/fs/bt_sec_db";
```

Comment:
May we use a configuration parameter ?

Reply:
you mean for the example or for the feature as a whole?